tools/x-docs/.gitignore

```diff
@@ -1,9 +1,4 @@
-# Project dependencies
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 .cache
-node_modules
 bower_components
```

Comment:
Move to top level?

Reply:
i didn't want to encourage this sort of thing